packages/typescript-resolver-files-e2e/src/test-modules/modules/topic/resolvers/Topic.ts

```diff
@@ -1,9 +1,6 @@
 import type { TopicResolvers } from './../../types.generated';
 export const Topic: TopicResolvers = {
   /* Implement Topic resolver logic here */
-  createdAt: () => {
-    /* Topic.createdAt resolver is required because Topic.createdAt's type is "union:string | Date" but TopicMapper.createdAt's type is "classOrInterface:Date" */
-  },
```

Comment:
This was not cleaned up correctly because previous rimraf command didn't clean up files in `test-modules/resolvers/` correctly